3ds/mbedtls/PKGBUILD

```diff
@@ -10,12 +11,11 @@ license=('apache')
 options=(!strip libtool staticlibs)
 depends=(3ds-zlib)
 makedepends=('3ds-pkg-config' 'devkitpro-pkgbuild-helpers')
-source=( 'https://tls.mbed.org/download/mbedtls-2.7.0-apache.tgz'
+source=( "https://tls.mbed.org/download/mbedtls-${pkgver}-apache.tgz"
          'libmbedtls-2.7.0.patch' )
```

Comment:
If this patch applies clean (i.e. doesn't fuzz to find the right lines) then rename it & use ${pkgver} for this too.

If it needs to fuzz then generate a new patch for 2.8.0